queryviews/models.py

```diff
@@ -47,9 +47,12 @@ class Meta:
 def get_count(model, filters):
     name = get_matview_name(model, filters)
     app_label, model_name = model.split('.')
-    count_model = queries[app_label][model_name][name + '__count']
+    try:
+        count_model = queries[app_label][model_name][name + '__count']
 
-    return count_model.objects.first().count
+        return count_model.objects.first().count
+    except KeyError:
+        return get_query(model, filters).count()
```

Comment:
Maybe a comment here: 'Couldn't find a specific materialized view, so do a slower query'?

Also, I wonder if it would be valuable to stick a time the `get_query` call and log it with `logging.warning` or something, so we could get some metrics from Papertrail about how bad these queries are?